### ISSUE TITLE
handle-done-callback should ignore skipped tests

### ISSUE THREAD

**Issue by gabegorelick** [lo1tuma/eslint-plugin-mocha#210]
`handle-done-callback` should ignore skipped tests, either via an option or by default.

Currently, this type of code fails:
```js
it.skip('test name', function (done) {
  // TODO finish this test
});
```
